microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -638,4 +638,13 @@ Issued: 23 October 2024
 
 {product-title} release 4.14.39 is now available. The list of enhancements and bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2024:8237[RHBA-2024:8237] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:8235[RHSA-2024:8235] advisory.
 
+For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-14-40-dp_{context}"]
+=== RHBA-2024:8699 - {microshift-short} 4.14.40 bug fix and security update advisory
+
+Issued: 7 November 2024
```

Comment:
```suggestion
Issued: 8 November 2024
```
Live now, so you can just update the date and and we can merge Monday.

Reply:
done.